apm-server.yml

```diff
@@ -382,25 +382,29 @@ apm-server:
       # Default value is `true`.
       #require_policy: true
 
-      # The configured event types, policies and rollover_aliases will be merged with
-      # the default setup. Only configure the mappings that should be customized.
-      # NOTICE: when customizing `rollover_alias`, it is strongly recommended to keep
-      # the prefix "apm-%{[observer.version]}" unchanged;
-      # in case the prefix is changed, `setup.template.name` and `setup.template.pattern`
-      # need to be changed accordingly to match the created indices
+      # The configured event types, policies and index suffixes will be merged with the default setup.
+      # Only configure the mappings that should be customized.
+      # Index names will always start with `apm-%{[observer.version]}-%{[event.type]}, e.g. apm-7.9.0-span.
```

Comment:
I find the phrasing in all this section a bit confusing/convoluted. 
Can't we just say that index are named like `apm-%{[observer.version]}-%{[event.type]}-{mapping.index_suffix}`, where `{mapping.index_suffix}` is the setting bellow and it is optional?
Or something like that, I don't know. Just an idea. Or maybe seek assistance from professionals...

Also confused about "Only configure the mappings that should be customized": not sure if that is trivial or if I am missing something.
Also the last note, "ensure they are aligned with the ILM indices": what does "aligned" mean? what are "ILM indices" (aren't they all ILM)? etc.

Reply:
I like Juan's idea here. What about something like this?

```
Customized mappings will be merged with the default setup, so you only need to configure mappings for the event types, policies, and index suffixes that you want to customize.
Indices are named in this way: `apm-%{[observer.version]}-%{[event.type]}-{index_suffix}`, e.g., apm-7.9.0-span-custom*.
The `index_suffix` is optional.
```

Reply:
Thank you both, I changed the wording.